src/components/DetailDialog/index.jsx

```diff
@@ -310,7 +310,7 @@ const DetailDialog = ({ serviceId, location, navigate }) => {
                           <strong>{sched.weekday}s</strong>
                         </td>
                         <td>
-                          {twelveHourTime(sched.opens_at)}—
+                          {twelveHourTime(sched.opens_at)} to{" "}
```

Comment:
@jhackett1 I think prettier added the `{" "}`, which feels a bit odd - is there a better way we could do this?